libafl/src/bolts/shmem.rs

```diff
@@ -588,6 +593,10 @@ pub mod unix_shmem {
             pub __glibc_reserved5: c_ulong,
         }
 
+        #[cfg(target_vendor = "apple")]
+        const MAX_MMAP_FILENAME_LEN: usize = 32;
```

Comment:
This is macOS's limit

Reply:
Maybe add this information as comment? Also, it's 31 chars according to the link you posted, not 32?

Reply:
fixed